File: Code/App/v2/Blood/ViewModels/BloodExaminationViewModel.cs
using BusinessLogic.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Blood.ViewModels
{
    public class BloodExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public BloodExaminationCommentViewModel BloodComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}

File: Code/App/separateDB/Blood/Scripts/App/Models/BloodExaminationViewModel.js
function BloodExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.BloodComment = new
        BloodExaminationCommentViewModel(data.BloodComment);
    this.examinationDescriptionBlood = ko.observable();

};